api-docs/README.md

```diff
@@ -0,0 +1,131 @@
+# Browser and Node.js analytics for Snowplow
```

Comment:
Do we need this README file? Seems to just duplicate the one in the root.

Reply:
Nope, this was from a prev version 👍